backend/lib/endpoints/posts.js

```diff
@@ -258,46 +257,53 @@ async function routes(app) {
         throw app.httpErrors.notFound();
       }
 
-      // TODO: add pagination
-      const [commentErr, commentQuery] = await app.to(
-        Comment.aggregate([
-          {
-            $match: {
-              parentId: null,
-              postId: mongoose.Types.ObjectId(postId),
+      const [comment] = await app.to(Comment.findOne({ postId }));
```

Comment:
Why this query before getting the comments? Is it to check if there are any comments at this post?

Reply:
Okay, this will work.
There is probably a way of doing this check at the same query to avoid 2 requests to the DB, but I will open a ticket and look into that, later on, to not block this now